examples/general/basic_dead_store/Cargo.toml

```
@@ -0,0 +1,20 @@
+[package]
+name = "basic_dead_store"
+version = "0.1.0"
+authors = ["Filipe Casal <fcasal@users.noreply.github.com>"]
+description = "A lint to find simple instances of dead stores in arrays."
+edition = "2021"
+publish = false
+
+[lib]
+crate-type = ["cdylib"]
```

```suggestion
crate-type = ["cdylib", "rlib"]
```
Both are needed, the latter for when all of the "general" lints are put into one `general` library.